themes/default/content/docs/concepts/vs/terraform/_index.md

```diff
@@ -92,11 +92,11 @@ Here is a summary of the key differences between Pulumi and Terraform:
 | ------- | ------ | --------- |
 | [Language Support](#language) | Python, TypeScript, JavaScript, Go, C#, F#, Java, YAML | HashiCorp Configuration Language (HCL) |
 | [IDE Support](#ide) | Code completion, strong typing, error squiggles, rich resource documentation, etc. | Limited |
+| [OSS License](#license) | Apache License 2.0 | Business Source License 1.1 |
```

Comment:
Nit, BUSL is not an "OSS License", so the row label doesn't work perfectly. Maybe:

```
[Open Source] | Yes, Apache 2.0 | No, Business Source License 1.1 |
```